src/client/client4.js

```diff
@@ -2521,29 +2521,17 @@ export default class Client4 {
         );
     };
 
-    // Role Routes
-
-    getRole = async (roleId) => {
-        return this.doFetch(
-            `${this.getRolesRoute()}/${roleId}`,
-            {method: 'get'}
-        );
-    };
-
-    getRoleByName = async (roleName) => {
```

Comment:
Sam here. Please add these back as it is good to have these as every API we have still has a corresponding client4 driver.